autogen/variables.tf

```diff
@@ -360,6 +360,18 @@ variable "pod_security_policy_config" {
   }]
 }
 
+variable "resource_usage_export_dataset_id" {
+  type        = string
+  description = "The dataset id of network egress metering for this cluster."
+  default     = ""
```

Comment:
Are you sure null is not more appropriate?

Reply:
No, because its compared to an empty string